neutrona/README.md

```diff
@@ -0,0 +1,50 @@
+# Agent Check: Neutrona
+
+## Overview
+
+This check monitors [Neutrona][1] cloud connectivity services to:
+ - Azure (ExpressRoute)
+
+## Setup
+
+### Installation
+
+The Neutrona check is not included in the [Datadog Agent][2] package, so you
+need to install it.
+
+Copy the check files manually or run the `_install.sh_` script if you are running the [Datadog Agent][2] on Debian/Ubuntu. This copies both the check script and sample configuration file and **restarts** the [Datadog Agent][2]. 
+
+
+### Configuration
+
+1. Edit the `neutrona.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+   Agent's configuration directory to start collecting your neutrona performance data.
+   See the [sample neutrona.d/conf.yaml][2] for all available configuration options.
+
+2. [Restart the Agent][3].
+
+### Validation
+
+[Run the Agent's status subcommand][4] and look for `neutrona` under the Checks section.
+
+## Data Collected
+
+See [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/neutrona/metadata.csv) for a list of metrics provided by this integration.
```

Comment:
https://github.com/DataDog/integrations-core/blob/master/neutrona/metadata.csv should be a referenced link below:

[metadata.csv][6]
...
[6]: https://github.com/DataDog/integrations-core/blob/master/neutrona/metadata.csv